setup.py

```diff
@@ -155,7 +155,7 @@
         "grpcio-testing>=1.56.2,<2",
         "minio==7.1.0",
         "mock==2.0.0",
-        "moto",
+        "moto<5",
```

Comment:
5 changed around the api and current tests do not pass without modification.

Reply:
I also stumbled into this, any idea why CI isn't failing as well?

Reply:
I *suspect* that some transitive dependency was previously holding it back.

Reply:
Moto v5 just released last week though, that's why CI succeeded